.vscode/arduino.json

```diff
@@ -0,0 +1,5 @@
+{
+    "board": "AZ3166:stm32f4:MXCHIP_AZ3166",
+    "configuration": "upload_method=OpenOCDMethod",
+    "port": "COM3"
+}
```

Comment:
Add new lines here. If you are in vscode the new lines at the end of the file will be added automatically.